Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.yml

```diff
@@ -530,27 +654,67 @@ script:
     - contextPath: URL.RecordedFuture.LastSeen
       description: Risk last seen timestamp
       type: date
-    description: Returns threat intelligence information for a URL in Recorded Future.
-  - name: recorded-future-get-threats-hash
-    arguments:
-    - name: detailed
-      auto: PREDEFINED
+    - contextPath: URL.Tags
+      description: Tags that are associated with the Domain
+      type: String
+    - contextPath: URL.CommunityNotes.Note
+      description: '...'
```

Comment:
A summary note on the mentioning of the URL in a source monitored by Recorded Future.